Formula/vault.rb

```diff
@@ -34,6 +34,39 @@ def install
     end
   end
 
+  plist_options :manual => "valt server -dev"
```

Comment:
Vault*?

Reply:
good catch, this is the display string and missed the typo